app/controllers/exercises_controller.rb

```diff
@@ -25,16 +25,19 @@ def create
     end
   end
 
-  def mark_as_finished
-    user_exercise = UserExercise.find_by(user_id: current_user.id, exercise_id: params[:exercise_id])
-    user_exercise.update_attribute(:status, "finished")
-    render nothing: true
+  def submit
+    @exercise = Exercise.find(params[:exercise_id]);
```

Comment:
don't need the semi-colon at the end, JavaScript is ruining your syntax :trollface: